go/ct/spc/specification.go

```diff
@@ -503,27 +503,34 @@ func getAllRules() []Rule {
 		{revision: R09_Berlin, warm: true, config: gen.StorageModified, gasCost: 2900},
 		{revision: R09_Berlin, warm: true, config: gen.StorageModifiedDeleted, gasCost: 100, gasRefund: 15000},
 		{revision: R09_Berlin, warm: true, config: gen.StorageModifiedRestored, gasCost: 100, gasRefund: 2800},
+	}
 
-		// {revision: R10_London, warm: false, config: gen.StorageAssigned, gasCost: 2200}, // invalid
-		{revision: R10_London, warm: false, config: gen.StorageAdded, gasCost: 22100},
-		// {revision: R10_London, warm: false, config: gen.StorageAddedDeleted, gasCost: 2200, gasRefund: 19900},  // invalid
-		// {revision: R10_London, warm: false, config: gen.StorageDeletedRestored, gasCost: 2200, gasRefund: 100}, // invalid
-		// {revision: R10_London, warm: false, config: gen.StorageDeletedAdded, gasCost: 2200, gasRefund: -4800},  // invalid
-		{revision: R10_London, warm: false, config: gen.StorageDeleted, gasCost: 5000, gasRefund: 4800},
-		{revision: R10_London, warm: false, config: gen.StorageModified, gasCost: 5000},
-		// {revision: R10_London, warm: false, config: gen.StorageModifiedDeleted, gasCost: 2200, gasRefund: 4800},  // invalid
-		// {revision: R10_London, warm: false, config: gen.StorageModifiedRestored, gasCost: 2200, gasRefund: 4900}, // invalid
-
-		{revision: R10_London, warm: true, config: gen.StorageAssigned, gasCost: 100},
-		{revision: R10_London, warm: true, config: gen.StorageAdded, gasCost: 20000},
-		{revision: R10_London, warm: true, config: gen.StorageAddedDeleted, gasCost: 100, gasRefund: 19900},
-		{revision: R10_London, warm: true, config: gen.StorageDeletedRestored, gasCost: 100, gasRefund: -2000},
-		{revision: R10_London, warm: true, config: gen.StorageDeletedAdded, gasCost: 100, gasRefund: -4800},
-		{revision: R10_London, warm: true, config: gen.StorageDeleted, gasCost: 2900, gasRefund: 4800},
-		{revision: R10_London, warm: true, config: gen.StorageModified, gasCost: 2900},
-		{revision: R10_London, warm: true, config: gen.StorageModifiedDeleted, gasCost: 100, gasRefund: 4800},
-		{revision: R10_London, warm: true, config: gen.StorageModifiedRestored, gasCost: 100, gasRefund: 2800},
+	for rev := R10_London; rev <= R12_Shanghai; rev++ {
```

Comment:
Would it be wrong to include `Cancun` already? Maybe even even using `NewestSupportedRevision` as the upper end?